src/host/twitter/hooks.ts

```diff
@@ -77,3 +83,13 @@ export const useTwitterExternalLinksPooling = () => {
 
   return { results };
 };
+
+export const useTwitterLocationInfo = () => {
+  const location = useLocation();
+
+  const isTwitter = isTwitterHostname(location.hostname ?? '');
+  const isTwitterHandlePage = isTwitterHandlePathname(location.pathname ?? '');
+  const isTwitterHomePage = isTwitterHomePathname(location.pathname ?? '');
+
+  return { isTwitter, isTwitterHandlePage, isTwitterHomePage };
+};
```

Comment:
can you reuse this in our other apps? There should be few todo's for that

Reply:
sure, I've found that in the `wallet-connect-modal.component.tsx` we also need to use that hook. Should we move it to the Shared module?

![image](https://github.com/idriss-crypto/browser-extensions/assets/106707979/8f97413f-171c-4348-afd7-7dd27944ac28)

Reply:
@SolutionsEngineer let's not move it to shared, wallet-connect-modal case is very specific, we only check it because coinbase wallet is having problems on Twitter after newest update, ideally they fix it on their side and we don't need to worry about it. There should be eslint-disable, I've it on my fresh PR, not sure if it wasn't on main before and I just added it now.

<img width="524" alt="image" src="https://github.com/idriss-crypto/browser-extensions/assets/50832972/f5081ca5-3f43-4058-a625-2e4062a5ee8b">

If they don't fix it and we want to get rid of this problem that this place is breaking our rules we might decide to inject te list of banned wallets from higher layer which is allowed to use host module, something like this:
<img width="683" alt="image" src="https://github.com/idriss-crypto/browser-extensions/assets/50832972/14a5f147-0218-4145-84c3-844b3677c268">